src/components/inline_edit/inline_edit_form.test.tsx

```diff
@@ -0,0 +1,303 @@
+    });
+
+    it('renders editModeProps.inputProps', () => {
+      const { container, getByTestSubject } = render(
+        <EuiInlineEditForm
+          {...commonInlineEditFormProps}
+          startWithEditOpen={true}
+          editModeProps={{
+            inputProps: {
+              prepend: 'Prepend Example',
+              'data-test-subj': 'customInput',
+            },
+          }}
+        />
+      );
+
+      expect(container.firstChild).toMatchSnapshot();
+      expect(getByTestSubject('customInput')).toBeTruthy();
+    });
+
+    it('renders editModeProps.formRowProps', () => {
+      const { container, getByTestSubject } = render(
+        <EuiInlineEditForm
+          {...commonInlineEditFormProps}
+          startWithEditOpen={true}
+          editModeProps={{
+            formRowProps: {
+              error: ['This is an error'],
+              'data-test-subj': 'customErrorText',
+            },
+          }}
+        />
+      );
+
+      expect(container.firstChild).toMatchSnapshot();
+      expect(getByTestSubject('customErrorText')).toBeTruthy();
+    });
+
+    it('renders save button and cancel button aria-labels', () => {
+      const { container, getByLabelText } = render(
+        <EuiInlineEditForm
+          {...commonInlineEditFormProps}
+          startWithEditOpen={true}
+          saveButtonAriaLabel="Yes! Let's save."
+          cancelButtonAriaLabel="Uh no. Do not save."
+        />
+      );
+
+      expect(container.firstChild).toMatchSnapshot();
+      expect(getByLabelText("Yes! Let's save.")).toBeTruthy();
+      expect(getByLabelText('Uh no. Do not save.')).toBeTruthy();
+    });
+
+    it('renders EuiSkeletonRectangles in place of editMode buttons when loading', () => {
+      const { container, getByTestSubject, queryByTestSubject } = render(
+        <EuiInlineEditForm
+          {...commonInlineEditFormProps}
+          startWithEditOpen={true}
+          isLoading={true}
+        />
+      );
+
+      expect(container.firstChild).toMatchSnapshot();
+
+      expect(getByTestSubject('euiInlineEditModeSaveLoading')).toBeTruthy();
+      expect(getByTestSubject('euiInlineEditModeCancelLoading')).toBeTruthy();
+
+      expect(queryByTestSubject('euiInlineEditModeSaveButton')).toBeFalsy();
+      expect(queryByTestSubject('euiInlineEditModeCancelButton')).toBeFalsy();
+    });
+
+    it('disables the save button when input is invalid ', () => {
+      const { container, getByTestSubject } = render(
+        <EuiInlineEditForm
+          {...commonInlineEditFormProps}
+          startWithEditOpen={true}
+          isInvalid={true}
+        />
+      );
+
+      expect(container.firstChild).toMatchSnapshot();
+
+      expect(
+        getByTestSubject('euiInlineEditModeInput').hasAttribute('aria-invalid')
+      ).toBeTruthy();
+
+      expect(getByTestSubject('euiInlineEditModeSaveButton')).toBeDisabled();
+      expect(getByTestSubject('euiInlineEditModeCancelButton')).toBeTruthy();
```

Comment:
Is this assertion all that helpful for this specific prop that we're testing? Should we instead check that the cancel button is not disabled?
```suggestion
      expect(getByTestSubject('euiInlineEditModeCancelButton')).not.toBeDisabled();
```